### ISSUE TITLE
Bug: Released version of frames validator uses outdated @xmtp/proto package

### ISSUE THREAD

**Issue by stephancill** [xmtp/xmtp-node-js-tools#226]
### Describe the bug

When installing @xmtp/frames-validator, the protobuf package that it uses is `"@xmtp/proto": "3.45.0"`, which is outdated. Please release the latest version

### Expected behavior

The main branch has the latest protobuf version:  `"@xmtp/proto": "3.56.0"`, but the installed package has `"@xmtp/proto": "3.45.0"`

**Comment by fabriguespe** [comment]
Thank you @stephancill ! For more context adding the original request

We are encountering some problems with the debugger that failing with the latest frames-validator  library. Works up to 0.5.0 Im sharing some of the logs we found 

```
Error: Mismatched state
     at checkUntrustedData (webpack-internal:///(rsc)/./node_modules/@xmtp/frames-validator/dist/index.js:179:15)
     at validateFramesPost (webpack-internal:///(rsc)/./node_modules/@xmtp/frames-validator/dist/index.js:141:11)
     at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
     at async getXmtpFrameMessage (webpack-internal:///(rsc)/./node_modules/frames.js/dist/xmtp/index.js:13:49)
     at async Object.getFrameMessage (webpack-internal:///(rsc)/./app/frames/frames.ts:42:36)
     ...
```

and this error msg now is generated from the debugger on xmtp protocol, going two click deep in the state, for example clicking "Learn more" -> click home button, or clicking "Send" -> input an address -> click "Next".
the payload that causes this message, is this one and I've checked and it is a Valid xmtp payload

```
payload: {
  mockData: {
    enabled: true,
    requesterFollowsCaster: false,
    casterFollowsRequester: false,
    likedCast: false,
    recastedCast: false
  },
  clientProtocol: 'xmtp@2024-02-09',
  untrustedData: {
    buttonIndex: 1,
    opaqueConversationIdentifier: '+rb4/6oBvD+OeEBwE4TQ+snEm6ZHo3qVmVvdXDTbcxk=',
    walletAddress: '0x310936Bd141525Df7BFE98a3d13102E7d5Ea96f8',
    inputText: '0x699d04F9994f181F3E310F70cF6aC8E8445aCe9A',
    url: 'http://localhost:3000/',
    timestamp: 1715206175482,
    unixTimestamp: 1715206175482
  },
  trustedData: {
    messageBytes: 'CkYKRApAa8e899ZapxMSeu989T5dvIzDhktfGmFL6Ggg+aJiQkR4y0fYceeSYMmQcLvJ76HJK3hXmvs+mI95v5yhGK5d9BABErACCpYBCkwIsbn0yvMxGkMKQQQUvlsRpcC/dfBwRdSIQeIRjiVDslTUfnCBLprgids+EPvNVDDXfvCbqu9SPBdFSQzlHz5JFj64XLIEFIGdPM6mEkYSRApAJxVj8/JXDuv8e9cIuYxqW+/SuGk7qmiufVIjZfqkGiM/S0mLtO2TZ7gPjm90tL262I+zuQtWCU4shpALEVD22RABEpQBCkwI/M70yvMxGkMKQQSG/0l8UwCf5DcC3vPp56j/Yf5G1RoywsVdKqXA40JDuXxnuJPBXXYUXKVxsd+fPpQjam2l1lB5xFEJ5KEbWRDhEkQKQgpAPTs0UXQJnkWW+kxSIOYAgY8igYOymgYDAN+G1gxh0FQmNSH8PcxiSIlEwZXHdzTh3pP0GBuXV6xCeNrtxoUk5xqSAQoWaHR0cDovL2xvY2FsaG9zdDozMDAwLxABGPr1hNL1MSIsK3JiNC82b0J2RCtPZUVCd0U0VFErc25FbTZaSG8zcVZtVnZkWERUYmN4az0o+vWE0gUyKjB4Njk5ZDA0Rjk5OTRmMTgxRjNFMzEwRjcwY0Y2YUM4RTg0NDVhQ2U5QToPeyJwYWdlSW5kZXgiOjB9'
  }
}
```

**Comment by rygine** [comment]
resolved by #232 and #233
